smac/acquisition/maximizer/local_search.py

```diff
@@ -187,7 +191,7 @@ def _get_init_points_from_previous_configs(
         previous_configs: list[Configuration]
             Previous configuration (e.g., from the runhistory).
         n_points: int
-            Number of initial points to be generated.
+            Number of initial points to be generated; selected from previous configs (+ random configs if necessary).
```

Comment:
But here no random configs are added right? And `n_points` is ignored?

Reply:
N_points is essentially the number of configs that are taken from previous_configs (the best), so e.g. for the random and local search 10, and then 10 random ones are added via the additional configs. Random configs are added if n_points > len(previous_configs) to have enough points.

Reply:
ahh okay